anesthetic/samples.py

```diff
@@ -89,6 +89,10 @@ def __init__(self, *args, **kwargs):
             self.root = None
             super(MCMCSamples, self).__init__(*args, **kwargs)
 
+            for param in self.columns:
+                if param not in self.limits:
+                    self.limits[param] = (self[param].min(), self[param].max())
```

Comment:
Fairly simple implementation. Too simple? (Am going to write tests, now)

I first was wondering whether these lines should go into `ChainReader` or `GetDistReader`, but ultimately decided that this doesn't actually belong in the readers. The readers _read_ (duh!) from the files, whereas this here is a post reading process. 

Am open to suggestions for a better placement in the project structure, though.

Reply:
This does indeed seem too easy, but as far as I can tell does seem to do what we want.